pkg/patch/cmd.go

```diff
@@ -68,13 +70,11 @@ func NewPatchCmd() *cobra.Command {
 	flags.BoolVar(&ua.ignoreError, "ignore-errors", false, "Ignore errors and continue patching")
 	flags.StringVarP(&ua.format, "format", "f", "openvex", "Output format, defaults to 'openvex'")
 	flags.StringVarP(&ua.output, "output", "o", "", "Output file path")
+	flags.BoolVar(&ua.updateAll, "update-all", false, "Update all packages without scanner input")
```

Comment:
can we add validation so users can't specifcy this with scanner and report?

Reply:
removed flag, just checking for optional report flag now